week-1/Homework/mandatory/4-tax.js

```diff
@@ -17,7 +24,14 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function formatCurrency() {}
+function formatCurrency(productPrice) {
```

Comment:
- I like the way you make `£` a variable on its own first.
- The way you named the variable `SalesPrice` doesn't follow convention. Can you work out what you are doing wrong here?
- `formatCurrency` tests are failing. That means your function implementation is wrong. You are close but missing important functionality that was requested in the comments: `prices must have 2 decimal places`

Reply:
camelCase :), you are right. I have also added .toFixed function to have 2 digit decimal added to the product price. Hope I have done the right thing.